source/Mission.cpp

```diff
@@ -1277,6 +1358,12 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 	subs["<commodity>"] = result.cargo;
 	subs["<tons>"] = to_string(result.cargoSize) + (result.cargoSize == 1 ? " ton" : " tons");
 	subs["<cargo>"] = subs["<tons>"] + " of " + subs["<commodity>"];
+	subs["<outfit>"] = result.OutfitName();
```

Comment:
Again, I am pretty sure you do not need the function OutfitName() when you can directly access the given variable. Which makes some of the functions around this new functionality obsolote. I dont really mind them though, as they still can become useful later

Reply:
OutfitName dereferences outfit with a guard statement if outfit is a null pointer.

Reply:
oh, right, should have noticed that